File: bridge/python/pipe.py
import json, sys
from {{ .OpFile }} import {{ .Method }}

if __name__ == '__main__':
    while True:
        request_json = sys.stdin.readline()
        if not request_json:
            break
        response_json = json.dumps({{ .Method }}(json.loads(request_json)))
        print response_json
        sys.stdout.flush()

File: bridge/ruby/pipe.rb
require "readline"
require "json"
require_relative "{{ .OpFile }}"

if __FILE__ == $0
    loop do
        request_json = STDIN.gets
        response_json = JSON.dump({{ .Method }}(JSON.parse(request_json)))
        STDOUT.puts response_json
        STDOUT.flush
    end
end
